<?xml version='1.0'?>
<ns2:test-suite xmlns:ns2='urn:model.allure.qatools.yandex.ru' start='1601592716587' stop='1601592810569'>
    <name>Paid blocks</name>
    <title>Paid blocks</title>
    <test-cases>
        <test-case start='1601592721996' status='failed' stop='1601592749661'>
            <name>MailChimp Block</name>
            <title>MailChimp Block</title>
            <labels/>
            <parameters/>
            <steps>
                <step start='1601592721997' status='undefined'>
                    <name>Can visit the block editor and add a MailChimp block</name>
                    <title>Can visit the block editor and add a MailChimp block</title>
                    <attachments>
                        <attachment title='Test failed: Paid blocks :: MailChimp Block' source='c27a4e50-0017-47dd-8330-0a6ad3f1d025-attachment.png' type='image/png' size='17194'/>
                    </attachments>
                    <steps/>
                </step>
            </steps>
            <attachments/>
            <failure>
                <message>TypeError: output.split is not a function</message>
                <stack-trace>TypeError: output.split is not a function
                    at split (/home/travis/build/Automattic/jetpack/tests/e2e/lib/plan-helper.js:29:16)
                    at tryCatch (/home/travis/build/Automattic/jetpack/node_modules/regenerator-runtime/runtime.js:45:40)
                    at Generator.invoke [as _invoke] (/home/travis/build/Automattic/jetpack/node_modules/regenerator-runtime/runtime.js:274:22)
                    at Generator.prototype.&lt;computed> [as next] (/home/travis/build/Automattic/jetpack/node_modules/regenerator-runtime/runtime.js:97:21)
                    at asyncGeneratorStep (/home/travis/build/Automattic/jetpack/node_modules/@babel/runtime/helpers/asyncToGenerator.js:3:24)
                    at _next (/home/travis/build/Automattic/jetpack/node_modules/@babel/runtime/helpers/asyncToGenerator.js:25:9)
                    at processTicksAndRejections (internal/process/task_queues.js:97:5)</stack-trace>
            </failure>
        </test-case>
        <test-case start='1601592749661' status='failed' stop='1601592777230'>
            <name>Pay with PayPal</name>
            <title>Pay with PayPal</title>
            <labels/>
            <parameters/>
            <steps>
                <step start='1601592749662' status='undefined'>
                    <name>Can visit the block editor and add a Pay with PayPal block</name>
                    <title>Can visit the block editor and add a Pay with PayPal block</title>
                    <attachments>
                        <attachment title='Test failed: Paid blocks :: Pay with PayPal' source='ef856782-ebef-46e2-9d78-8b2da66efb6b-attachment.png' type='image/png' size='17194'/>
                    </attachments>
                    <steps/>
                </step>
            </steps>
            <attachments/>
            <failure>
                <message>TypeError: output.split is not a function</message>
                <stack-trace>TypeError: output.split is not a function
                    at split (/home/travis/build/Automattic/jetpack/tests/e2e/lib/plan-helper.js:29:16)
                    at tryCatch (/home/travis/build/Automattic/jetpack/node_modules/regenerator-runtime/runtime.js:45:40)
                    at Generator.invoke [as _invoke] (/home/travis/build/Automattic/jetpack/node_modules/regenerator-runtime/runtime.js:274:22)
                    at Generator.prototype.&lt;computed> [as next] (/home/travis/build/Automattic/jetpack/node_modules/regenerator-runtime/runtime.js:97:21)
                    at asyncGeneratorStep (/home/travis/build/Automattic/jetpack/node_modules/@babel/runtime/helpers/asyncToGenerator.js:3:24)
                    at _next (/home/travis/build/Automattic/jetpack/node_modules/@babel/runtime/helpers/asyncToGenerator.js:25:9)
                    at processTicksAndRejections (internal/process/task_queues.js:97:5)</stack-trace>
            </failure>
        </test-case>
        <test-case start='1601592777230' status='failed' stop='1601592804861'>
            <name>WordAds block</name>
            <title>WordAds block</title>
            <labels/>
            <parameters/>
            <steps>
                <step start='1601592777231' status='undefined'>
                    <name>Can visit the block editor and add a WordAds block</name>
                    <title>Can visit the block editor and add a WordAds block</title>
                    <attachments>
                        <attachment title='Test failed: Paid blocks :: WordAds block' source='de5a83f2-aa70-4c7e-b6eb-d1fbf0b7a793-attachment.png' type='image/png' size='17194'/>
                    </attachments>
                    <steps/>
                </step>
            </steps>
            <attachments/>
            <failure>
                <message>TypeError: output.split is not a function</message>
                <stack-trace>TypeError: output.split is not a function
                    at split (/home/travis/build/Automattic/jetpack/tests/e2e/lib/plan-helper.js:29:16)
                    at tryCatch (/home/travis/build/Automattic/jetpack/node_modules/regenerator-runtime/runtime.js:45:40)
                    at Generator.invoke [as _invoke] (/home/travis/build/Automattic/jetpack/node_modules/regenerator-runtime/runtime.js:274:22)
                    at Generator.prototype.&lt;computed> [as next] (/home/travis/build/Automattic/jetpack/node_modules/regenerator-runtime/runtime.js:97:21)
                    at asyncGeneratorStep (/home/travis/build/Automattic/jetpack/node_modules/@babel/runtime/helpers/asyncToGenerator.js:3:24)
                    at _next (/home/travis/build/Automattic/jetpack/node_modules/@babel/runtime/helpers/asyncToGenerator.js:25:9)
                    at processTicksAndRejections (internal/process/task_queues.js:97:5)</stack-trace>
            </failure>
        </test-case>
    </test-cases>
</ns2:test-suite>